plugins/vendor/zoom/lib/model/kZoomEventHanlder.php

```diff
@@ -43,6 +43,9 @@ public function processEvent($event)
 		switch($event->eventType)
 		{
 			case kEventType::RECORDING_VIDEO_COMPLETED:
+			case kEventType::RECORDING_TRANSCRIPT_COMPLETED:
+				KalturaLog::notice('This is an old Zoom event type - Not processing');
```

Comment:
Make sure that in such session we would. have info what was the event and what was the data, if that info is missing in the session logs, than add it here.

Reply:
Done, the event data would be printed at:
`	/**
	 * @action recordingComplete
	 * @throws Exception
	 */
	public function recordingCompleteAction()
	{
		KalturaResponseCacher::disableCache();
		myPartnerUtils::resetAllFilters();
		$kZoomEventHandler = new kZoomEventHanlder(self::getZoomConfiguration());
		$event = $kZoomEventHandler->parseEvent(); - log event data 
		$zoomIntegration = ZoomHelper::getZoomIntegrationByAccountId($event->accountId);
		ZoomHelper::verifyZoomIntegration($zoomIntegration);
		$this->setPartnerFilters($zoomIntegration->getPartnerId());
		$kZoomEventHandler->processEvent($event); - skip processing
	}`